docker-compose.yml

```diff
@@ -3,7 +3,7 @@ version: "3.5"
 services:
   web:
     image: ${docker_repo:-zokradonh}/kopano_web
-    container_name: web
+    container_name: ${COMPOSE_PROJECT_NAME}-web
     restart: unless-stopped
     ports:
       - "2015:2015"
```

Comment:
if the goal is to run this multiple times on the same hos, this hardcoded port should probably go into a variable as well.

Reply:
True, did not occur to me since I ran my stack without the web-service. Why is this port exposed anyway?

Reply:
A fallback (it's the default port for caddy) it's also useful and less confusing when running web without ssl)